microsoft/testsuites/gpu/gpusuite.py

```diff
@@ -247,14 +246,33 @@ def verify_gpu_cuda_with_pytorch(
         _install_driver(node, log_path, log)
         _check_driver_installed(node, log)
 
-        _install_cudnn(node)
+        # pytorch and CUDA needs 4GB space to download and install
+        free_partition = node.find_partition_with_freespace(5)
+        work_path = free_partition + "/python_packages"
+        node.tools[Mkdir].create_directory(work_path, sudo=True)
 
-        gpu = node.features[Gpu]
+        node.execute(
+            f"chmod -R 777 {work_path};",
+            shell=True,
+            sudo=True,
+        )
+        # work_path = free_partition + "/lisawork"
+        log.debug(f"Current work dir is {work_path}")
+
+        _install_cudnn(node, log, work_path)
 
         pip = node.tools[Pip]
+        # pip.install_packages("torch")
+
         if not pip.exists_package("torch"):
-            pip.install_packages("torch")
+            pip.install_packages("torch", work_path)
+            # node.execute(
+            #     f"pip3 install torch -q --cache-dir={work_path} -b {work_path}",
+            #     shell=True,
+            #     sudo=True,
+            # )
 
+        gpu = node.features[Gpu]
         gpu_script = "import torch;print(f'gpu count: {torch.cuda.device_count()}')"
```

Comment:
I haven't seen the new package path is included in PythonPath environment variable, how to make sure it works?

Reply:
Previously I don't add "--target" in the command, which means it will install to the default folder.
In the next change I will add PythonPath